components/card/card.js

```diff
@@ -1 +1,27 @@
-export function createCharacterCard() {}
+const cardContainer = document.querySelector('[data-js="card-container"]');
+export function createCharacterCard(character) {
```

Comment:
As you only export this function from this file, we usually use `export default` for it.
```suggestion
export default function createCharacterCard(character) {
```